qualification/report/katdoc.sty

```diff
@@ -1,64 +1,84 @@
-% File: katdoc.sty
-% Subject: Stylesheet for KAT document
-% Author: M. Welz and A. Pinska; based on template by R.T. Lord and J.C. Guzman
 \ProvidesPackage{katdoc}
-% \setcounter{errorcontextlines}{999}
-
-\newcommand{\stylesheetVersion}{1.1.2-dirty}
+\newcommand{\stylesheetVersion}{1.1.3-DSP}
 
+\usepackage[utf8]{inputenc}
 \usepackage{graphicx}
 \usepackage{lastpage}
 \usepackage{mathptmx}
 \usepackage[T1]{fontenc}
 \usepackage{geometry}
 \usepackage{array}
 \usepackage{babel}
+\usepackage{csquotes}  % Latex recommends this when babel is used
 \usepackage{fancyhdr}
 \usepackage{tabularx}
 \usepackage{longtable}
 \usepackage{multirow}
 \usepackage{ifthen}
 \usepackage{float}
 \usepackage{enumitem}
+\usepackage{mathtools}
+\usepackage{tablefootnote}
+\usepackage{color}
+\usepackage{colortbl}
 
 % For plotting to work nicely.
-\usepackage[utf8]{inputenc}
+\usepackage{pgf}
 \usepackage{pgfplots}
 \DeclareUnicodeCharacter{2212}{−}
 \usepgfplotslibrary{groupplots,dateplot}
 \usetikzlibrary{patterns,shapes.arrows}
 \pgfplotsset{compat=newest}
 
-\geometry{verbose,a4paper,tmargin=32mm,bmargin=27mm,lmargin=21mm,rmargin=19mm,headheight=10mm,headsep=12mm,footskip=12mm}
-\pagestyle{empty}
+\geometry{nomarginpar,verbose,a4paper,tmargin=32mm,bmargin=27mm,lmargin=21mm,rmargin=19mm,headheight=10mm,headsep=12mm,footskip=12mm}
 \setlength{\parskip}{\medskipamount}
 \setlength{\parindent}{0pt}
-\pagenumbering{arabic}
```

Comment:
I'm guessing this is removed because it's the default?

Reply:
No, because the preamble pages are numbered with roman numerals. It changes back to arabic later on (line 238 as I look at it at the moment)